traffic_ops/traffic_ops_golang/deliveryservice/deliveryservices.go

```diff
@@ -858,6 +859,7 @@ func updateV30(w http.ResponseWriter, r *http.Request, inf *api.APIInfo, ds *tc.
 		&ds.InnerHeaderRewrite,
 		&ds.LastHeaderRewrite,
 		&ds.ServiceCategory,
+		&ds.MaxRequestHeaderSize,
```

Comment:
The way the DS API works, we also need to query the existing value of `max_request_header_size` in `updateV15`, in order for 1.5 requests to be properly _upgraded_ to 3.0 requests. Otherwise, a pre-3.0 request would blow away the existing value.

The [README](https://github.com/apache/trafficcontrol/blob/master/traffic_ops/traffic_ops_golang/README.md#updating-a-minor-version) explains this a bit better, but might be out of date now that `DeliveryServiceNullable` is deprecated :(